ui/raidboss/data/06-ew/dungeon/another_aloalo_island.ts

```diff
@@ -0,0 +1,1550 @@
+          'front': output.right!(),
+          'back': output.left!(),
+          'left': output.front!(),
+          'right': output.back!(),
+        }[map];
+      },
+      outputStrings: {
+        text: {
+          en: 'Mindhack',
+          de: 'Geistlenkung',
+          fr: 'Piratage mental', // FIXME
+          ja: '強制移動',
+          cn: '强制移动', // FIXME
+          ko: '강제이동', // FIXME
+        },
+        front: {
+          en: 'Mindhack: Forward',
+          de: 'Geistlenkung: Vorwärts',
+          fr: 'Piratage mental : Vers l\'avant',
+          ja: '強制移動 : 前',
+          cn: '强制移动 : 前',
+          ko: '강제이동: 앞',
+        },
+        back: {
+          en: 'Mindhack: Back',
+          de: 'Geistlenkung: Rückwärts',
+          fr: 'Piratage mental : Vers l\'arrière',
+          ja: '強制移動 : 後ろ',
+          cn: '强制移动 : 后',
+          ko: '강제이동: 뒤',
+        },
+        left: {
+          en: 'Mindhack: Left',
+          de: 'Geistlenkung: Links',
+          fr: 'Piratage mental : Vers la gauche',
+          ja: '強制移動 : 左',
+          cn: '强制移动 : 左',
+          ko: '강제이동: 왼쪽',
+        },
+        right: {
+          en: 'Mindhack: Right',
+          de: 'Geistlenkung: Rechts',
+          fr: 'Piratage mental : Vers la droite',
+          ja: '強制移動 : 右',
+          cn: '强制移动 : 右',
+          ko: '강제이동: 오른쪽',
+        },
+      },
+    },
+    {
+      id: 'AAI Lala Spatial Tactics',
+      type: 'Ability',
+      netRegex: { id: '88A0', source: 'Lala', capture: false },
+      delaySeconds: 1.2,
+      durationSeconds: 20,
+      infoText: (data, _matches, output) => {
+        if (data.lalaSubtractive === undefined)
+          return;
+        return output.text!({ num: data.lalaSubtractive });
+      },
+      outputStrings: {
+        text: {
+          en: '${num}',
+          ja: 'カウント: ${num}',
+        },
+      },
+    },
+    {
+      id: 'AAI Lala Symmetric Surge Partner',
+      type: 'Ability',
+      netRegex: { id: '88A1', source: 'Lala', capture: false },
+      delaySeconds: 1.2,
+      alertText: (data, _matches, output) => {
+        const [s1, s2] = data.gainList.filter((x) => x.effectId === 'E8B').map((x) => x.target);
+        if (s1 === undefined || s2 === undefined)
+          return;
+
+        const dps1 = data.party.isDPS(s1);
+        const dps2 = data.party.isDPS(s2);
+        const same = (dps1 && dps2) || (!dps1 && !dps2);
+        if (same)
+          return;
+
+        if (data.role === 'dps')
+          return output.swap!();
+      },
+      outputStrings: {
+        swap: {
+          en: 'DPS swap partner!',
+          ja: 'DPS同志に入れ替え！',
+        },
+      },
+    },
+    {
+      id: 'AAI Lala Symmetric Surge',
+      type: 'Ability',
+      netRegex: { id: '88A1', source: 'Lala', capture: false },
+      delaySeconds: 31,
+      durationSeconds: 5,
+      response: Responses.getTogether(),
+      run: (data) => data.gainList = [],
+    },
+    {
+      id: 'AAI Lala Arcane Plot Spread',
```

Comment:
```suggestion
      id: 'AAI Lala Arcane Point Spread',
```
the spread damage here is from Arcane Point, Arcane Plot is what puts down the floor arrow markers.